src/actions/compileContract.ts

```diff
@@ -60,7 +60,7 @@ async function actuallyCompileContract(
 
   updateContractStatus("compiling");
   const solangResult = await runCommand([
-    "../clones/solang/target/release/solang",
+    "/home/giannipop/root/pendulum/solang/target/release/solang",
```

Comment:
Was this change intended?

Reply:
Sorry that is a mistake. Every time it runs in local I need to point to my local solang. Will delete now.